.github/workflows/fossa.yml

```diff
@@ -0,0 +1,27 @@
+name: FOSSA
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: "^1.14.x"
+      - run: go version
+      # Runs a set of commands to initialize and analyze with FOSSA
+      - name: run FOSSA analysis
+        env:
+          # FOSSA Push-Only API Token
+          FOSSA_API_KEY: '5cda1cd0d5036f6f368682c23a5fdfec'
```

Comment:
I think this should go into the SMI org secrets store in GitHub and referenced here with `${{ secrets. FOSSA_API_KEY }}`

Reply:
@stefanprodan GitHub Actions do not support passing secrets from the forks, and if people will submit PR from their forks (what they typically do), this won't work :( See this as a reference - https://github.com/kedacore/keda/pull/860; we've discovered this issue there.

However, this API key is safe to be exposed publicly (it was designed for this purpose) - https://docs.fossa.com/docs/api-reference#push-only-api-token

Reply:
Ah I totally forgot about that GH limitation...